.gitignore

```diff
@@ -27,13 +27,6 @@
 /DOCS/man/mpv.pdf
 /DOCS/man/mpv.toc
 
-/waf
-/waf-*
-/waf3-*
-/build
```

Comment:
Pretty sure you don't want to nuke this one.

Reply:
Why not?

Reply:
Well I dunno what everyone does but I would guess most people name their build directory "build" (it's in the example too).

Reply:
Meson puts a `.gitignore` in all its build files. https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

Reply:
Oh, I did not even know that... Well nevermind then.

Reply:
Yeah, the idea is that you should not need to maintain a gitignore that doesn't even cover all the things people actually call it...

I always use "builddir", many people suffix their build directory name with -release or -debug or various other things, etc.

You cannot really keep track of this with a gitignore checked into git, but meson *can* do it automatically for you. :D

Reply:
(This is one of the reasons why meson > waf. :wink:)